mitmproxy/contentviews/image/pillow.py

```diff
@@ -19,6 +22,11 @@ class ViewImage(base.View):
     ]
 
     def __call__(self, data, **metadata):
+        if imghdr.what('', h=data) == 'png':
```

Comment:
I removed the empty string and - `TypeError: what() missing 1 required positional argument: 'file'`